app/javascript/controllers/submission_details_controller.js

```diff
@@ -7,20 +7,16 @@ export default class extends Controller {
   eligibleCheck(e) {
     if (e.target.checked) {
       this.judgingStatusHiddenTarget.value = "selected"
-      this.winnerCheckboxTarget.disabled = false
```

Comment:
these checkboxes shouldn't toggle their disabled status when the other input is checked/unchecked anymore. instead some other state needs to change on the submission after the status is saved.